src/App/PropertyPythonObject.cpp

```diff
@@ -87,6 +87,7 @@ std::string PropertyPythonObject::toString() const
             throw Py::Exception();
         Py::Callable method(pickle.getAttr(std::string("dumps")));
         Py::Object dump;
+#if PY_VERSION_HEX < 0x03b0000

```

Comment:
I wonder why this change would be needed. For Python features that implement the __getstate__/__setstate__ functions persistence might be broken.

Reply:
Mistaken commit, intended elsewhere.